DEVELOPMENT.md

```diff
@@ -151,6 +151,83 @@ to start a debugger to which attach from your IDE.
    curl localhost:8081/apis/apps/v1/deployments
    ```
 
+## Database
+
+### Database requirements
+
+* psql
+    ```bash
+    $ sudo dnf install postgresql
+    $ psql --version
+    psql (PostgreSQL) 15.6
+    ```
+
+* Optional: A SQL client is recommended to interact easily with the database.
+  For example [DBeaver Community](https://dbeaver.io/). To configure DBeaver, check these 
+  [instructions](https://github.com/kubearchive/kubearchive/blob/main/database/README.md).
+
+
+### Test the connection to the Database.
+  
+1.  Identify the PostgreSQL pod.
+    ```bash
+    $ kubectl get pods -n [namespace]
+    NAME                        READY   STATUS    RESTARTS   AGE
+    postgresql-65db968757-ncnvk   1/1     Running   0          2m38s
+    ```
+1.  Replace the `postgres_pod`, `db_user` and `db_name` with the values from `ps-secret.yaml`, or from the `values.yaml` file. The `--password` prompts for the password interactively.
+    ```bash
+    $ kubectl exec -it [postgres_pod] -n [namespace] -- psql -h localhost -U [db_user] --password -p 5432 [db_name]
+    ```
+    ```bash
+    $ kubectl exec -it postgresql-65db968757-ncnvk -n kubearchive -- psql -h localhost -U ps_user --password -p 5432 postgresdb
+    Password: 
+    psql (16.3 (Debian 16.3-1.pgdg120+1))
+    Type "help" for help.
+
+    postgresdb=# 
+    ```
```

Comment:
I think we need to remove this whole section. The connection to the database should be working out of the box. If we see that there is some common error, we can add it to the known issues.

Reply:
Done